src/components/backtotop.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import fontawesome from '@fortawesome/fontawesome';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import faArrowUp from '@fortawesome/fontawesome-free-solid/faArrowUp';
+
+fontawesome.library.add(faArrowUp);
+
+const BackToTop = () => {
+  const handleBackToTop = () => {
+    window.scroll({ top: 0, behavior: 'smooth' });
+  };
+  
+  return (
+    <div className="back-to-top" onClick={ () => handleBackToTop() } onScroll={ () => alert() } >
```

Comment:
What does `onScroll={() => alert()}` do? Do we need it?

Also super nitpicky format thing, can we organize the div like 

<div
	className...